script.js

```diff
@@ -0,0 +1,35 @@
+// Mouse Gun №8
+// const body = document.querySelector('body');
+// body.addEventListener('click', function (e) {
+//     const elem = document.elementFromPoint(e.clientX, e.clientY);
+//     elem.style.visibility = "hidden";
+//     console.log(`X = ${e.clientX} Y = ${e.clientY}`);
+// });
+
+
+// Mouse Gun №9
+const body = document.querySelector('body');
+let elem ;
+
+const listener = function (event) {
+    const coordinates = {
+        left : event.clientX - elem.clientWidth / 2,
+        top : event.clientY - elem.clientHeight / 2
+    };
+    elem.style.position = 'absolute';
```

Comment:
is bad to use global variables from closure, refactor to avoid it. Ask me if you cannot find solution